addons/nuxeo-spreadsheet/ftest/features/step_definitions/spreadsheet.js

```diff
@@ -0,0 +1,68 @@
+import { Then, When } from 'cucumber';
+import Spreadsheet from '../../pages/spreadsheet';
+
+When('I open the spreadsheet', function() {
+  const button = this.ui.results.actions.element('nuxeo-spreadsheet-button').click();
+
+  const dialog = button.element('#dialog');
+  dialog.waitForVisible();
```

Comment:
nit: Since we don't use the dialog, we could just do:
```suggestion
  button.waitForVisible('#dialog');
```